src/App.js

```diff
@@ -20,7 +19,6 @@ const App = () => {
         <Route path='/navigation' element={<Navigation />} />
         <Route path='/forms' element={<Forms />} />
         <Route path='/' element={<HomePage />} />
```

Comment:
Please put this HomePage route to the top of route paths

Reply:
Put Route HomePage on top